bin/publish_build

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+echo -e "CURRENT_BRANCH: ${TRAVIS_BRANCH}\n"
+echo -e "PULL_REQUEST: ${TRAVIS_PULL_REQUEST}\n"
+
+# build the distribution files first
+$(npm bin)/grunt build
+
+git config --global user.email "antonio@brandwatch.com"
+git config --global user.name "Antonio D'Ettole"
+
+# This specifies the repository we are going to work with.
+DIST_REPO_SLUG="BrandwatchLtd/selleckt-builds"
+
+# This specifies the user who is associated to the GH_TOKEN
+USER="codazzo"
+
+# This ensure that no directories within dist will be copied when script is run.
+INCLUDED_FILES=`find dist -maxdepth 1 -type f`
+
+# The branch the builds are pushed ot
+RELEASE_BRANCH="master"
+
+LATEST_TAG=`git describe --tags`
+LATEST_COMMIT_TAG=`git describe --tags --abbrev=0`
+
+echo -e "DIST_REPO_SLUG: ${DIST_REPO_SLUG}\n"
+echo -e "INCLUDED_FILES: ${INCLUDED_FILES}\n"
+echo -e "CURRENT_BRANCH: ${TRAVIS_BRANCH}\n"
+
+if [[ $TRAVIS_PULL_REQUEST != "false" ]]; then
+  echo "not publishing because this is a pull request."
+  exit 0
+fi
+
+if [[ $TRAVIS_BRANCH != "master" ]]; then
+  echo "not publishing because this is not a build of the master branch."
+  exit 0
+fi
+
+if [[ -z $GH_TOKEN ]]; then
+  echo "secure environment variables not detected."
+  echo "not a repo owner, exiting."
+  exit 0
+fi
+
+# sending output to /dev/null to prevent GH_TOKEN leak on error
+git clone https://${USER}:${GH_TOKEN}@github.com/${DIST_REPO_SLUG}.git bower_selleckt &> /dev/null
+cp -r ${INCLUDED_FILES} bower_selleckt/
```

Comment:
Should this wipe out any additional files in the destination dir too?  You could use `rsync` with `--delete` for the copy if so (and `--exclude` anything that needs to stay)